src/todo-class.js

```diff
@@ -0,0 +1,10 @@
+export default class Todo {
+    static count = 0;
+
+    constructor(completed, description) {
+      // eslint-disable-next-line
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.

Reply:
Thanks!